week6/src/Store.java

```diff
@@ -0,0 +1,202 @@
+
+                            //recording the sale
+                            salesList.add(new Sales(animalProducts.get(animalElement).getProduct(), animalProducts.get(animalElement).getPrice()));
+                            String animalConfirmed = animalProducts.get(animalElement).getProduct();
+
+                            //removing sold product from inventory
+                            animalProducts.remove(animalElement);
+                            System.out.println("You successfully sold the " + animalConfirmed + "! Press [Enter] to continue...");
+                            try{System.in.read();}
+                            catch(Exception e){}
+                            break;
+
+                        case 2:
+                            //vegetal product sale selection
+                            System.out.println("Press [Enter] and select the vegetal product you want to sell from the list using [0] [1] [2] [3]...");
+                            try{System.in.read();}
+                            catch(Exception e){}
+                            vegetalProducts.forEach(System.out::println);
+                            int vegetalElement = sc.nextInt();
+
+                            //recording the sale
+                            salesList.add(new Sales(vegetalProducts.get(vegetalElement).getProduct(),vegetalProducts.get(vegetalElement).getPrice()));
+                            String vegetalConfirmed = vegetalProducts.get(vegetalElement).getProduct();
+
+                            //removing sold product from inventory
+                            vegetalProducts.remove(vegetalElement);
+                            System.out.println("You successfully sold the " + vegetalConfirmed + "! Press [Enter] to continue...");
+                            try{System.in.read();}
+                            catch(Exception e){}
+                            break;
+                        default:
+                            System.out.println("Input Error! Press [Enter] to continue...");
+                            try{System.in.read();}
+                            catch(Exception e){}
+                            break;
+                    }
+                    break;
+
+                case 3:
+                    //checks if list is empty before proceeding
+                    if (salesList.isEmpty()) {
+                        System.out.println("You haven't sold anything today. Press Enter to continue...");
+                        try{System.in.read();}
+                        catch(Exception e){}
+
+                        //totaling sales
+                    } else {
+                        double total = 0;
+                        for(int i =0; i < salesList.size(); i++) {
+                            total = total + salesList.get(i).getPrice();
+                        }
+
+                        //printing sale report
+                        SimpleDateFormat t = new SimpleDateFormat("dd/MM/yyyy");
```

Comment:
you didn't give the user the chance to tell you which date you need to show the report for.
You need to store the sale date in the Sales object.